lib/OpenQA/Worker/Cache.pm

```diff
@@ -90,9 +78,13 @@ sub init {
     log_info(__PACKAGE__ . ': loading database from ' . $self->db_file);
     $self->dsn("sqlite:" . $self->db_file);
     $self->deploy_cache unless -e $self->db_file;
+    eval { $self->sqlite->migrations->name('cache_service')->from_data->migrate };
+    log_error "Deploying DB failed: $@" if $@;
```

Comment:
I understand that you want to improve step-by-step. But shouldn't we stop the execution completely here to prevent any further error messages and strange behavior which would only cause confusion? And maybe suggest that deleting the existing SQLite database would fix this issue.

Reply:
Sounds like a good idea.

Reply:
And done.